packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -287,8 +313,12 @@ class MemoryBodyState extends State<MemoryBody>
             }
 
             eventChartController?.zoomDuration = duration;
```

Comment:
shouldn't setting the zoomDuration trigger `dirty = true` instead of having to remember to set it from outside the controller?

Reply:
Done.